DB/index.js

```diff
@@ -6,7 +6,7 @@ const ENV = require('../.env');
 
 const mongoPass = ENV.MONGODB_PASS;
 
-const DB_URI = `mongodb+srv://root:${mongoPass}@gameon.p78bo.mongodb.net/?retryWrites=true&w=majority`;
+const DB_URI = `mongodb+srv://royce:${mongoPass}@gameon.p78bo.mongodb.net/?retryWrites=true&w=majority`;
```

Comment:
is everyone's user going to be **_royce_**?
seems like this should be a variable that should go in your env if it changes for each of you